js/src/UXCapture.js

```diff
@@ -2,44 +2,51 @@ import ExpectedMark from "./ExpectedMark";
 import View from "./View";
 
 const NOOP = () => {};
+const INTERACTION_START_MARK = "interactionStart";
+const NAVIGATION_START_MARK = "navigationStart";
```

Comment:
nitpick, I like 'noun-adjective' naming to make relationships/namespacing more clear (similar to PCV selector naming in CSS) - e.g. these could be

```js
MARK_START_INTERACTION
MARK_START_NAVIGATION
```

or

```
const MARKS = {
  interactionStart: 'interactionStart',
  navigationStart: 'navigationStart',
}
```

not a blocker, something to consider

Reply:
Any reason for interactionStart name rather than transitionStart?